From aedec75c772919fa77c33854d3389d8ca73c25bf Mon Sep 17 00:00:00 2001
From: Alison Wang <b18965@freescale.com>
Date: Thu, 4 Aug 2011 09:59:54 +0800
Subject: [PATCH 45/52] Add high-resolution kernel timer support

This patch fixed the bug that high-resolution kernel timer
could not be enabled. ARCH_USES_GETTIMEOFFSET is disabled.

Signed-off-by: Alison Wang <b18965@freescale.com>
---
 arch/m68k/Kconfig |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

--- a/arch/m68k/Kconfig
+++ b/arch/m68k/Kconfig
@@ -64,7 +64,7 @@ config HZ
 	default 100
 
 config ARCH_USES_GETTIMEOFFSET
-	def_bool y
+	def_bool n
 
 source "init/Kconfig"
 
